### DISCUSSION TITLE
Creating the cron file .cron extension not working in Ubuntu

### DISCUSSION THREAD

**Discussion by marcbenedi** [neilmunday/slurm-mail#19] [Q&A] [answered]
Greetings,

As far as I could test in my Ubuntu machine, creating the cron file `/etc/cron.d/slurm-mail.cron` as suggested in the `README.md` won't work.

I had to remove the `.cron` extension of the file, then `cron` would execute it.

Can anyone confirm this? :)

Thanks!

**Comment by neilmunday** [comment, marked as answer]
Hi,

I've only tested Slurm-Mail under RHEL and CentOS.

I will update the README to remove the extension as it's not a requirement.

Thanks for the info.

Cheers,

Neil.
